package.json

```diff
@@ -19,9 +23,27 @@
     "Paul Boocock"
   ],
   "license": "Apache-2.0",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/snowplow-incubator/snowplow-react-native-tracker"
+  },
   "peerDependencies": {
-    "react-native": ">=0.59.8",
-    "react": ">=16.8.6"
+    "react": ">=16.8.6",
+    "react-native": ">=0.59.8"
   },
-  "dependencies": {}
+  "devDependencies": {
+    "@babel/core": "^7.14.5",
```

Comment:
Thank you for reviewing @paulboocock !
Indeed, i had blindly left `babel` around. Now removed it completely as we use `tsc` for compilation. Also left the declarations to [dts-bundle-generator](https://github.com/timocov/dts-bundle-generator) (thanks for the pointer!) so that they now get all bundled into `index.d.ts`.
For ref, the last force-push also removed the redundant strict options from `tsconfig.json`, exported the interfaces and types, added links to Readme, removed quoting object keys and switched to `import {createTracker} ..` both in readme examples and DemoApp.